metadata.json

```diff
@@ -1,24 +1,31 @@
 {
   "name": "duosecurity-duo_unix",
-  "version": "0.3.3",
+  "version": "0.3.4",
   "author": "Duo Security",
   "summary": "Installs, configures, and manages Duo Unix.",
   "license": "GPL-2.0",
-  "source": "https://github.com/duosecurity/puppet-duo_unix",
+  "source": "https://github.com/mozilla-it/puppet-duo_unix",
   "project_page": "https://www.duosecurity.com/docs/duounix",
-  "issues_url": "https://github.com/duosecurity/puppet-duo_unix/issues",
+  "issues_url": "",
+  "description": "Duo Unix Puppet module.",
+  "dependencies": [
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.25.0 < 7.0.0"
+    }
+  ],
   "operatingsystem_support": [
     {
       "operatingsystem": "RedHat",
-      "operatingsystemrelease": [ "6", "7" ]
+      "operatingsystemrelease": [ "6", "7", "8" ]
     },
     {
       "operatingsystem": "CentOS",
-      "operatingsystemrelease": [ "5", "6", "7" ]
+      "operatingsystemrelease": [ "5", "6", "7", "8" ]
```

Comment:
We have no means or intention of testing 5 or 6, and they're all but EOL.  Let's kick them out.